slack_rules/slack_idp_configuration_change.py

```diff
@@ -0,0 +1,21 @@
+from panther_base_helpers import slack_alert_context
+
+IDP_CHANGE_ACTIONS ={
+    "idp_configuration_added": "Slack IDP Configuration Added",
+    "idp_configuration_deleted": "Slack IDP Configuration Deleted",
+    "idp_prod_configuration_updated": "Slack IDP Configuration Updated",
+}
+
+
+def rule(event):
+    return event.get("action") in IDP_CHANGE_ACTIONS
+
+
+def title(event):
+    if event.get("action") in IDP_CHANGE_ACTIONS:
+        return IDP_CHANGE_ACTIONS.get(event.get("action"))
+    return "Slack IDP Configuration Changed"
```

Comment:
With the `rule()` being falsey for non IDP_CHANGE_ACTIONS, would line 17 ever be reached? ( or would it matter if it is reached if line 11's always False? )

Reply:
This is a fallback to reduce friction in case customers changed the logic